File: Uri/1013.py

x = raw_input()
y = x.split(' ')

a = int(y[0])
b = int(y[1])
c = int(y[2])

m = (a+b+abs(a-b))/2
m2 = (m+c+abs(m-c))/2

print m2,"eh o maior"

File: Uri/1013.c
#include<stdio.h>
#include<stdlib.h>
#include<math.h>

int main(){
    int a, b, c, mab, mbc, mac;
    scanf("%d %d %d", &a, &b, &c);
    mab = (a + b + abs(a - b))/2;
    mbc = (b + c + abs(b - c))/2;
    mac = (a + c + abs(a - c))/2;
    if ((mab == a && mbc == b) || (mac == a && mbc == c)){
        printf("%d eh o maior\n", a);
    }
    else if ((mab == b && mac == a) || (mbc == b && mac == c)){
        printf("%d eh o maior\n", b);
    }
    else {
        printf("%d eh o maior\n", c);
    }
    return 0;
}
